apps/andi/lib/andi/input_schemas/input_converter.ex

```diff
@@ -72,18 +69,29 @@ defmodule Andi.InputSchemas.InputConverter do
     Map.take(map, DatasetInput.technical_keys())
   end
 
-  defp keyword_string_to_list(nil), do: []
-  defp keyword_string_to_list(""), do: []
+  defp keywords_to_list(nil), do: []
+  defp keywords_to_list(""), do: []
 
-  defp keyword_string_to_list(keywords) do
+  defp keywords_to_list(keywords) when is_binary(keywords) do
     keywords
     |> String.split(", ")
     |> Enum.map(&String.trim/1)
   end
 
+  defp keywords_to_list(keywords) when is_list(keywords), do: keywords
+
   defp date_to_iso8601_datetime(date) do
     time_const = "00:00:00Z"
 
     "#{Date.to_iso8601(date)} #{time_const}"
   end
+
+  defp fix_modified_date(map) do
```

Comment:
I still think it would be cleaner to make `modifiedDate` a string type and then add a validation that ensures it's date. But I could be missing something.

Reply:
The main issue we ran into is that I *think* the phoenix form is expecting this field to be a date, and we're basically converting everything down to the types required by the form in order to make it consistent.  We can verify that the date input on the form will/won't work with a string field in the changeset.

Reply:
Oh, I see